pkg/network/node/ovscontroller.go

```diff
@@ -223,8 +223,10 @@ func (oc *ovsController) NewTransaction() ovs.Transaction {
 	return oc.ovs.NewTransaction()
 }
 
-func (oc *ovsController) ensureOvsPort(hostVeth, sandboxID string) (int, error) {
-	return oc.ovs.AddPort(hostVeth, -1, "external-ids=sandbox="+sandboxID)
+func (oc *ovsController) ensureOvsPort(hostVeth, sandboxID, podIP string) (int, error) {
+	return oc.ovs.AddPort(hostVeth, -1,
+		fmt.Sprintf(`external-ids=sandbox="%s",ip="%s"`, sandboxID, podIP),
```

Comment:
Don't remember top off the head. Double check if this is 'external_ids=' or 'external_ids:', pretty sure its the colon in some usage, maybe not while creating the port.

Reply:
@rajatchopra IIRC it's = for setting, but : for finding (eg : is sub-match so you can find an entry by one external-id, while = is match all)

Reply:
+1

Reply:
It's not "set" vs "find" though, it's "value of the whole map" vs "value of a single element".

`set external-ids=foo=bar` means `external_ids = map[string]string{ "foo": "bar" }`
`set external-ids:foo=bar` means `external_ids["foo"] = "bar"` (other members are unchanged)
`find external-ids=foo=bar` means `if reflect.DeepEqual(external_ids, map[string]string{ "foo": "bar" }) { ...`
`find external-ids:foo=bar` means `if external_ids["foo"] == "bar" { ...`